### ISSUE TITLE
[feature] show discarded card

### ISSUE THREAD

**Issue by D1mon** [supertriodo/Arena-Tracker#67]
The discarded cards are displayed among the deck, but it shows which card was dropped.

When will this be useful? For example, I walked away and did not see what card was dropped, it will be displayed in the box window, but I do not know which one was dropped. Also, in the game itself, the discarded cards are not shown.

You also need to display and discarded enemy cards.

**Comment by supertriodo** [comment]
Can you rephrase it or add some screenshot, I'm really lost reading your post.

**Comment by D1mon** [comment]
when I or an opponent has 10 cards in hand, the eleventh card taken is discarted. It is necessary to show the received maps more than 10.

**Comment by supertriodo** [comment]
Part of this feature is solved, cards discarded from other cards are shown in replay tab. We could make the cards discarded because your hand is full to appear as drawn and discarded in the replay tab. I'll add it to the possible list of features to be voted by patrons 5$+
